spec.bs

```diff
@@ -1540,6 +1540,24 @@ trust that the merchant showed the user the correct amount in their checkout
 flow (and any fraud discoveries are post-payment, when the user checks their
 account statement).
 
+## Attacker generated BBKs ## {#sctn-security-attacker-generated-bbk}
+
+The bank can and should protect against attacker generated BBKs by
```

Comment:
I see we do not do this correctly in the 'Merchant-supplied authentication data' section already, but `s/bank/Relying Party` here (if desired, you can write "Relying Party (e.g. a bank)")